docs/overview/status/ProjectStatus.mdx

```diff
@@ -30,7 +30,7 @@ NEXT UI Kit supports the latest, stable releases of all major browsers and platf
 
 | Edge  | Firefox | Chrome | Safari (macOS) |     IE 11     |
 | :---: | :-----: | :----: | :------------: | :-----------: |
-| >= 91 |  >= 78  | >= 90  |     >= 14      | Not supported |
+|✅ |  ✅  | ✅  |     ✅      | Not supported |
```

Comment:
What does this mean?

Reply:
We haven't been very objective on the browser versions supported, neither doing specific tests.
Do you have a suggestion of what to put there?

Reply:
These versions are quite objective, but there indeed instances where we're not following them (eg. `@layer`)